### PR TITLE
T4832: dhcp: Add IPv6-only dhcp option support (RFC 8925)

### PR DESCRIPTION
**Pull request by vfreex**
Clients supporting this DHCP option (DHCP option 108, RFC 8925) will disable its IPv4 network stack for configured number of seconds and operate in IPv6-only mode.

Clients supporting this option include iOS 15+ and macOS 12.0.1+.

Example command:
```sh
set service dhcp-server shared-network-name LAN6 subnet 192.168.64.0/24 ipv6-only-preferred 0
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

* https://phabricator.vyos.net/T4832

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [x] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: interface-definitions/dhcp-server.xml.in [hunk 1]
```diff
@@ -373,6 +373,20 @@
...
+                      <valueHelp>
+                        <format>u32</format>
+                        <description>The number of seconds the client should disable DHCPv4 for</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="numeric" argument="--range 0-4294967295"/>
+                      </constraint>
+                      <constraintErrorMessage>IPv6-only wait time must be between 0 and 4294967295 (49 days)</constraintErrorMessage>
+                    </properties>
+                    <defaultValue>0</defaultValue>
```
Comment (c-po) [inline interface-definitions/dhcp-server.xml.in]: Why do we need a default value here? Simply using no default value should disable this feature.

File: interface-definitions/dhcp-server.xml.in [hunk 2]
```diff
@@ -373,6 +373,20 @@
                       </leafNode>
                     </children>
                   </tagNode >
+                  <leafNode name="ipv6-only-preferred">
+                    <properties>
+                      <help>Signal IPv6-only-capable hosts to disable the IPv4 stack on the host interface (RFC 8925)</help>
```
Comment (c-po) [inline interface-definitions/dhcp-server.xml.in]: This should read: "Disable IPv4 on IPv6 only hosts (RFC 8925)"

File: interface-definitions/dhcp-server.xml.in [hunk 3]
```diff
@@ -373,6 +373,20 @@
...
+                      <valueHelp>
+                        <format>u32</format>
+                        <description>The number of seconds the client should disable DHCPv4 for</description>
```
Comment (c-po) [inline interface-definitions/dhcp-server.xml.in]: This should simply read: "Seconds"

**Review by c-po — CHANGES_REQUESTED** [review]

File: interface-definitions/dhcp-server.xml.in (hunk 1 above)
Comment (vfreex) [reply interface-definitions/dhcp-server.xml.in]: Removed.

**Review by dmbaturin — APPROVED** [review]

**Review by c-po — APPROVED** [review]

**Pull request merged**